assets/userscript.js

```diff
@@ -101,22 +101,22 @@ let userscriptPromise;
   function createSwitch(text, enabled, onClick) {
     const container = document.createElement("div");
     container.style.marginBottom = "20px";
-    container.className = "labelRow-NnoUIp";
+    container.className = "labelRow__523f2";
```

Comment:
May be used consts for class names like this? It simplifies updates in future

Reply:
True, I could see that helping in the future even if Discord doesn't really re-roll their styling too often.

I think this is a little much for this PR alone though so I'll follow up in another once this is merged.